datacollection/openapi/ati/unstructured-data-upload.yaml

```diff
@@ -157,8 +160,16 @@ components:
             type: string
             maxLength: 254
             format: ".+@.+"
-
-
+          custom_field_names:
+            description: An optional field for capturing specific metadata for a data source. For example, weather data files require the specification of station name and point location. This array captures the field names. Content here requires content in the "custom_field_values" field.
```

Comment:
chg 'An optional field for capturing' to something like 'captures specific...

we want to state its datatype in the type: field

Reply:
Like I mention in the email, here's the JSON Schema for the custom data fields.

               {                                                               
                  "type": "array",                                              
                  "minItems": 1,                                                
                  "items": {                                                    
                    "type": "object",                                           
                    "required": [                                               
                      "field_name"                                              
                    ],                                                          
                    "properties": {                                             
                      "field_name": {                                           
                        "type": "string",                                       
                        "minLength": 1,                                         
                        "maxLength": 1000                                       
                      },                                                        
                      "field_value": {                                          
                        "type": "string",                                       
                        "minLength": 0,                                         
                        "maxLength": 1000                                       
                      }                                                         
                    },                                                          
                    "additionalProperties": false                               
                  }                                                             
                }

Reply:
I've added my changes to custom_fields to this branch, and Tim's original custom_fields is now labelled "outdated", and there is a single line showing the new commit at the bottom just elow this comment, labelled "af51bb9"